docs/winc-prerequisites.md

```diff
@@ -0,0 +1,40 @@
+# Windows Container Workloads Prerequisites
+Below are the prerequisites for Windows Container Workloads. Please also see the cloud provider-specific sections that 
+may be relevant.
+
+## Supported Kubernetes Distribution Version
+* OKD/OCP 4.6+ must be used in order to configure Windows VMs
+* OKD/OCP 4.8+ and WMCO 3.0+ must be used in order to configure Bring-Your-Own-Host(BYOH) Windows instances
+
+## Supported Cloud Providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+## Supported Windows Server versions
+The table below outlines the supported 
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the cloud
+platform you are using. *These are the only Windows Server versions supported by Windows Containers.* Any other Windows 
```

Comment:
I think we should choose between cloud provider and cloud platform to remain consistent here.